discord-stubs/iterators.pyi

```diff
@@ -22,7 +22,7 @@ class _AsyncIterator(Generic[_IT]):
         self, func: Callable[[_IT], Union[_IT, Coroutine[Any, Any, _IT]]]
     ) -> _MappedAsyncIterator[_IT]: ...
     def filter(
-        self, predicate: Callable[[_IT], Union[_IT, Coroutine[Any, Any, _IT]]]
+        self, predicate: Callable[[_IT], bool]
```

Comment:
The predicate can return a `bool` or a coroutine resolving in a `bool`, so this should be `Union[bool, Coroutine[Any, Any, bool]]`.

Reply:
Ah indeed, that implies find is wrong as well

Reply:
Yep. I was going to fix it after I merged your PR, but you took care of it. Thanks!! :)